README.md

```diff
@@ -70,6 +70,7 @@
 ## 🚀 Live Demo <a name="live-demo"></a>
```

Comment:
- [ ] As required for this project The project should be deployed properly, and all pages, links, and images should work as expected. but when click on the link you shared in your `README` you might be using the wrong `URL` This is the result we get :point_down:  this was mentioned in the previous review but I can see that you made some effort on updating it but we still have the same problem , knidly take care of this then ask for another review when you will be ready :wink: 

![wrongURL](https://user-images.githubusercontent.com/88894525/218196661-484936f5-e538-41dc-9446-59f91c227bd3.png)